src/dev/i18n/extract_default_translations.js

```diff
@@ -79,9 +78,8 @@ export function validateMessageNamespace(id, filePath) {
   );
 
   if (!id.startsWith(`${expectedNamespace}.`)) {
-    throw createFailError(`${chalk.white.bgRed(' I18N ERROR ')} \
-Expected "${id}" id to have "${expectedNamespace}" namespace. \
-See i18nrc.json for the list of supported namespaces.`);
+    throw createFailError(`Expected "${id}" id to have "${expectedNamespace}" namespace. \
+See .i18nrc.json for the list of supported namespaces.`);
   }
```

Comment:
__nit:__ Let's export `isFailError` in `src/dev/run/index.js` and do something like this in `extractMessagesFromPathToMap` so that we can preserve stack trace for unexpected errors:
```js
for (const { name, content } of files) {
  try {
    .......
  } catch (error) {
    if (isFailError(error)) {
      throw createFailError(
        `${chalk.white.bgRed(' I18N ERROR ')} Error in ${normalizePath(name)}\n${error}`
      );
    }

    throw error;
  }
}
```